indico/web/templates/_ical_export.html

```diff
@@ -8,6 +8,7 @@
 
     {% block extra_download %}{% endblock %}
 
+    {% block persistent_ui %}
```

Comment:
I did not indent the contents of this rather big block on purpose since this template or at least most of its content should become obsolete soon anyway.